app/src/main/java/com/architectcoders/arquitectomarvel/model/MarvelApiRest.kt

```diff
@@ -0,0 +1,23 @@
+package com.architectcoders.arquitectomarvel.model
+
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object MarvelApiRest {
+
+    private val okHttpClient = HttpLoggingInterceptor().run {
+        level = HttpLoggingInterceptor.Level.BODY
+        OkHttpClient.Builder().addInterceptor(this).build()
+    }
+
+    val service: ApiService = Retrofit.Builder()
+        .baseUrl(BASE_URL)
+        .client(okHttpClient)
+        .addConverterFactory(GsonConverterFactory.create())
```

Comment:
Why gson and not moshi? We can improve performance changing to moshi...

Reply:
Perferct. Eleazan I think like you. 
Moshi is better. We passed it then